### ISSUE TITLE
Invisibility issues with teams

### ISSUE THREAD

**Issue by Lilmac12312** [drtshock/HCFProblems#279]
When using 1.8 you are able to see those using invisibility potions who aren't in your faction. I'm unsure if this was meant to be or is a bug but can this be fixed?..

**Comment by nsporillo** [comment]
Duplicate of #172, #257 , #278.
Upstream bug + explanation: https://github.com/drtshock/Factions/issues/327

A fix appears to be possible using [ProtocolLib](http://dev.bukkit.org/bukkit-plugins/protocollib/forum/65015-how-to-set-invisiblity-potion-only-for-one-team/)...

I might investigate a fix by hooking into ProtocolLib, it is one of those things that relies entirely on another plugin for functionality, but it'd be a neat fix because I can see why this is kind of an issue..

**Comment by Lilmac12312** [comment]
Thank you, appreciate this a lot. Hopefully there is a positive outcome once this is done :)

**Comment by nsporillo** [comment]
![img_4940](https://cloud.githubusercontent.com/assets/1480197/8298102/37c25632-1932-11e5-9ac6-ae5a0b244e36.JPG)

This is the spec that i'm going to try and implement.

**Comment by nsporillo** [comment]
@Lilmac12312 spent the last few hours trying to figure this one out using ProtocolLib. Have not figured out any way to accomplish this...

**Comment by Lilmac12312** [comment]
@nsporillo damn man. Hopefully you can, best of luck.

**Comment by lyzha** [comment]
Good luck with that! That would be so cool if you could fix that odd bug

**Comment by nsporillo** [comment]
Dont think this is really fixable unfortunately. Instead of leaving it open to dry out, going to just close it now.
